### ISSUE TITLE
Disable top drag in "Show Panel Model" example

### ISSUE THREAD

**Issue by Sergionius** [SCENEE/FloatingPanel#409]
### Description

Hello thank you for great library! All we need now, it disable top drag if panel is full or half(if full is disabled). I tried in 1.7.3 version and it was ok
### Expected behavior
We drag to top and panel stay on it side or small jump to top
### Actual behavior
We can drag panel to top whatever we want
![ezgif-1-c8b373c50ccd](https://user-images.githubusercontent.com/10280032/100448669-b518f600-30c3-11eb-8f97-a417140cce4d.gif)

### Steps to reproduce
clone repository, open and run Samples project and choose Show Panel Model

### Environment

**Library version**
last

**Installation method**

* clone from GitHub
 
**iOS version(s)**
14.2

**Xcode version**
12.2

**Comment by SCENEE** [comment]
`{top,bottom}InteractionBuffer` properties are removed from FloatingPanelLayout since v2. So you need to define a panel boundary as follows.
```swift
func floatingPanelDidMove(_ vc: FloatingPanelController) {
    if vc.isAttracting == false {
        let loc = vc.surfaceLocation
        let minY = vc.surfaceLocation(for: .full).y - 6.0
        let maxY = vc.surfaceLocation(for: .tip).y + 6.0
        vc.surfaceLocation = CGPoint(x: loc.x, y: min(max(loc.y, minY), maxY))
    }
}]
```
From [README > Specify the panel move's boundary](https://github.com/SCENEE/FloatingPanel#specify-the-panel-moves-boundary)

**Comment by ravilich86** [comment]
It doesn't work when we move the panel with speed. When released, the panel jumps up for a while

**Comment by scenee** [comment]
Hi, @ravilich86 
If `vc.isAttracting == false` is removed, the panel doesn't jump up beyond the boundary after releasing it. However a floating panel would have no collision behavior so that it looks settle for a while at the top of the boundary.

**Comment by armanarutiunov** [comment]
@scenee thanks a lot for your work! 

I've also stumbled upon this issue and your workaround works for me. However, I'm wondering, is it possible to find a better solution? Ideally, as a user, I expect that after surfaceLocation passes the limit Y, the scrollView that we provide via `track(scrollView: UIScrollView)` would start scrolling instead. With your suggested workaround, it's possible to scroll the content only after the panel is at the `.full` state (in my case).

**Comment by scenee** [comment]
Good catch, @armanarutiunov! I wasn’t aware of the use case. Thank you for letting me know. I’m addressing #455 now, which looks related to your suggestion. I would consider it to improve the continuous scroll of a panel as well.
